definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -26,14 +26,16 @@
     "SecurityClassification": "Public"
   },
   {
-    "LiveFrom": "01/01/2017",
+    "LiveFrom": "04/08/2020",
     "CaseTypeID": "DIVORCE",
-    "ID": "servicePayment",
-    "Name": "Service payment exists",
-    "Description": "Service payment exists",
-    "DisplayOrder": 2,
-    "PreConditionState(s)": "AwaitingServicePayment",
+    "ID": "ConfirmServicePayment",
+    "Name": "Confirm service payment",
+    "Description": "Confirm service payment",
+    "DisplayOrder": 13,
+    "PreConditionState(s)": "AwaitingPayment",
     "PostConditionState": "AwaitingServiceConsideration",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/set-up-confirm-service-payment-event",
```

Comment:
why not `set-up-confirm-service-payment`?

It would be the only divorce endpoint with "event" in it. There is no point to have "event" in callback. Every single callback endpoint is for event.